content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -109,7 +109,8 @@ can be written manually or generated by user code.
 > `dvc.yaml`. Additionally, a `dvc.lock` file is also created or updated by
 > `dvc run` and `dvc repro`, to record the pipeline state.
 
-Here's a comprehensive `dvc.yaml` example:
+Here's a comprehensive example of a machine learning pipeline, described in
+`dvc.yaml`:
```

Comment:
Good intention here but I'd revert this one because the example is very specifically about the dvc.yaml format, even when the stages do imply some basic ML stuff is happening. Let's find better places to add the term in this doc.

Reply:
Probably at the very top of https://dvc.org/doc/user-guide/dvc-files-and-directories#dvcyaml-file, e.g.:

> ... describe ~~data~~ **data science or machine learning** pipelines

Reply:
OK, reverted it and applied suggestion. That's perfect... seems like you're OK with the more obvious placements so I'll try to do that more.

Reply:
Sounds good.